compose.yaml

```diff
@@ -4,19 +4,26 @@ services:
       context: .
       target: development
     ports:
-      - '3002:3002'
+      - '3000:3000'
       - '9002:9229'
     links:
       - 'redis:redis'
     depends_on:
       redis:
         condition: service_started
     environment:
-      PORT: 3002
+      PORT: 3000
       NODE_ENV: development
       ALLOW_ERROR_VIEWS: ${ALLOW_ERROR_VIEWS}
       DAL_ENDPOINT: ${DAL_ENDPOINT:-http://fcp-dal-api:3005/graphql}
-      DAL_CONNECTION: true
+      DAL_CONNECTION: false
```

Comment:
Would be good to set this to read from local environment like the others and default to false if not present.